build.sbt

```diff
@@ -0,0 +1,46 @@
+import sbt.Keys._
+import org.scalajs.sbtplugin.cross.CrossProject
+
+name := "scalajs-java-time"
+
+version := "0.0.1-SNAPSHOT"
+
+organization := "org.scala-js"
+
+scalaVersion := "2.11.7"
+
+scalacOptions ++= Seq("-deprecation", "-feature", "-Xfatal-warnings")
+
+homepage := Some(url("http://scala-js.org/"))
+
+licenses += ("MIT", url("http://opensource.org/licenses/mit-license.php"))
+
+testOptions += Tests.Argument(
+    TestFramework("com.novocode.junit.JUnitFramework"), "-v", "-a")
+
+lazy val root: Project = project.in(file(".")).
+  enablePlugins(ScalaJSPlugin).
+  enablePlugins(ScalaJSJUnitPlugin)
+
+lazy val testSuite = CrossProject(
```

Comment:
Consider using the `crossProject` macro (and don't really care if the id of the JS project is called `testSuiteJS` as a result)